app/views/dashboard/show.html.erb

```diff
@@ -1 +1,24 @@
 <h1>Dashboard</h1>
+<h2>Project timeline</h2>
+
+<ul>
+  <% @prospects.each do |prospect| %>
+    <li>
+      <%= prospect.first %>
```

Comment:
`<%= prospect.first %>` or `<%= prospect.second %>` look very cryptic. Can you address this issue? Maybe yield both things separately ?